docs/content/docs/concepts/signers.mdx

```diff
@@ -1,6 +1,73 @@
 ---
 title: Signers
-description: Sign transactions and messages however you like
+description: Accounts with signing capabilities
 ---
 
-TODO
+## Introduction
+
+Signers are an abstraction that combines an account with an implementation that computes signatures on behalf of that account. No matter what actually computes the signature – a wallet app, a network API, the `SubtleCrypto` API, or a userspace implementation – so long as it implements the correct signer interface for your intended purpose, you can use it with all of Kit's signer-aware functions.
+
+Signers are designed to make it easier to build, sign, and send transactions by taking the guesswork out of which accounts' key pairs need to sign a transaction. Signer-aware APIs allow you to associate signer objects with each transaction instruction that requires them, enabling Kit's transaction planning, signing, and sending functions to automatically collect and invoke them.
+
+## Installation
+
+Signers are **included within the `@solana/kit` library** but you may also install them using their standalone package.
+
+```package-install
+@solana/signers
+```
+
+## What is a signer?
+
+All signers are wrappers around an [`Address`](/api/type-aliases/Address). This means that most APIs that require an `Address` can be made to accept a signer. Each specific type of signer adds one or more capabilities, such as the ability to sign a message or a transaction on behalf of the account with that address. Some even add the ability to sign _and_ send a transaction, which is common for wallets that you use in your browser or on your phone.
+
+```ts twoslash
+// @noErrors: 2769
```

Comment:
Did this so that I don't have to bother adding a lifetime to the transaction message in the example.